ReSharper.FSharp/src/FSharp.Psi.Features/src/Daemon/Highlightings/Errors.xml

```diff
@@ -131,4 +131,11 @@
     <QuickFix>ReplaceAppExprWithArgFix</QuickFix>
   </Warning>
 
+  <Warning staticGroup="FSharpErrors" name="LambdaCanBeSimplified">
+    <Parameter type="ILambdaExpr" name="lambdaExpr"/>
+    <Parameter type="int" name="redundantArgsCount"/>
+    <Behavour attributeID="DEADCODE" overlapResolvePolicy="DEADCODE"/>
+    <Message value="Lambda can be simplified"/>
```

Comment:
Please don't use dead code highlighting here. For cases like the following one it breaks any useful highlighting:
```fsharp
fun a b -> foo 123 x.Prop b
```